services/state/roster/roster.proto

```diff
@@ -68,14 +79,20 @@ message RosterEntry {
     bytes gossip_ca_certificate = 3;
 
     /**
-     * The long term elliptic-curve cryptography
-     * public key for the node used in TSS.
+     * An ALT_BN128 elliptic curve public encryption key.<br/>
```

Comment:
Perhaps a little too specific here:
```suggestion
     * An elliptic curve public encryption key.<br/>
     * This is currently an ALT_BN128 key.<br/>
```
Also, we prefer not to have a paragraph break within the description for a field; so the tag on line 83 could be removed.